webpush/encrypt.go

```diff
@@ -55,12 +55,36 @@ import (
 	"strings"
 )
 
+// Encryption version.
+type Version int
+
+const (
+	// The version that is widely deployed with WebPush (as of 2016-11).
+	AESGCM = iota
+	// The most recent version, the salt, record size and key identifier
+	// are included in a header that is part of the encrypted content coding.
+	AES128GCM
+)
+
+func (v Version) String() string {
+	switch v {
+	case AESGCM:
+		return "aesgcm"
+	case AES128GCM:
+		return "aes128gcm"
+	}
+	return ""
```

Comment:
Is there ever a scenario with an empty string will actually be returned? My naivety to Go says this should thrown error rather than return an empty string.